p2p/protocol/identify/obsaddr.go

```diff
@@ -432,27 +434,89 @@ func (oas *ObservedAddrManager) recordObservationUnlocked(conn network.Conn, obs
 	oas.addrs[localString] = append(oas.addrs[localString], oa)
 }
 
-// 1. If we have an activated address, we are behind an easy NAT.
+// For a given transport Protocol (TCP/UDP):
+//
+// 1. If we have an activated address, we are behind an Cone NAT.
```

Comment:
@willscott 

As per the taxonomy, it can be a Full-Cone NAT, Port Restricted Cone NAT or a Restricted Cone NAT:

https://dh2i.com/kbs/kbs-2961448-understanding-different-nat-types-and-hole-punching/

I've combined all three into a "Cone NAT".

Reply:
haven't heard that term before, and found it unclear. expand comment to list all 3 possibilities?

Reply:
@willscott 

They're defined in RFC 3489. I've updated the docs here with more details and have listed all 3 possibilities.